eogrow/utils/testing.py

```diff
@@ -217,6 +209,19 @@ def _get_coords_sample(geom: Polygon | MultiPolygon | Any) -> list[tuple[float,
     return stats
 
 
+def _calculate_parquet_stats(data: pd.DataFrame, config: StatCalcConfig) -> JsonDict:
+    stats = {"columns": list(data), "row_count": len(data)}
+
+    if len(data):
+        subsample: gpd.GeoDataFrame = data.sample(min(len(data), config.num_random_values), random_state=42)
```

Comment:
why do you cast this as `gpd.GeoDataFrame` here? Couldn't we leave it as `pd.DataFrame`? Does it perhaps auto convert the wkb/wkt into geometries?

Reply:
because pandas/geopandas dont expose their types and mypy didnt catch that the copy-pasted annotation was wrong :)

Reply:
but it was just that, an annotation, didnt change the fact that it was a pandas dataframe